docs/guides/Guide-Nested.md

```diff
@@ -88,7 +88,7 @@ In this case, the NavigatorWrappingScreen is not a navigator, but it renders a n
 class NavigatorWrappingScreen extends React.Component {
   render() {
     return (
-      <View>
+      <View style={{flex: 1}}>
```

Comment:
I think this is an unnecessary change and only makes the example more verbose.

Reply:
Without flex: 1 then the screen is blank.
With flex:1 it works as described.
 react-navigation@v1.0.0-beta.12